core/nginx/conf/tls.conf

```diff
@@ -3,7 +3,6 @@ ssl_certificate_key {{ TLS[1] }};
 {% if TLS_FLAVOR in ['letsencrypt','mail-letsencrypt'] %}
 ssl_certificate {{ TLS[2] }};
 ssl_certificate_key {{ TLS[3] }};
-ssl_trusted_certificate /etc/ssl/certs/ca-cert-DST_Root_CA_X3.pem;
```

Comment:
This will break OSCP stapling... instead we should put /etc/ssl/certs/ca-cert-ISRG_Root_X1.pem